core/src/modules/cli.ts

```diff
@@ -196,17 +210,21 @@ export namespace GrouparooCLI {
       GrouparooCLI.logger.log("");
       GrouparooCLI.logger.log(cyanBold(formattedTitle));
       const headings = ["PROFILES", "SOURCES", "DESTINATIONS"];
+      if (finalSummaryLogs[3].length > 0) headings.push("WARNINGS");
```

Comment:
This might be a little brittle, but we can worry about that when we next add to this summary

Reply:
agree